app/src/main/res/layout/activity_main.xml

```diff
@@ -11,7 +11,13 @@
         android:id="@+id/text_baby"
         style="@style/MainMenu_Style"
         android:background="@android:color/holo_blue_dark"
-        android:text="Baby" />
+        android:text="Baby's information" />
+
+    <TextView
+        android:id="@+id/text_health"
+        style="@style/MainMenu_Style"
+        android:background="@android:color/darker_gray"
```

Comment:
it's fine to use an android resource, but prefer defining your own in the XML file, and pointing to the android resource from there instead